netutils/netinit/netinit.c

```diff
@@ -343,6 +365,85 @@ static void netinit_set_macaddr(void)
 #  define netinit_set_macaddr()
 #endif
 
+#ifdef CONFIG_BOARDCTL_NETCONF
+/****************************************************************************
```

Comment:
This should depend on the existence of etc/sysconfig/network-scripts/ifcfg. boardctl() should not be used as discussed in apache/incubator-nuttx#1886